internal/handlers/tigris/msg_getparameter.go

```diff
@@ -68,17 +70,18 @@ func (h *Handler) MsgGetParameter(ctx context.Context, msg *wire.OpMsg) (*wire.O
 }
 
 // selectParam is makes a selection of requested parameters.
-func selectParam(document, resDB *types.Document) (doc *types.Document, err error) {
+func selectParam(_, resDB *types.Document) (doc *types.Document, err error) {
 	doc = must.NotFail(types.NewDocument())
-	keys := document.Keys()
 
-	for _, k := range keys {
-		if k == "getParameter" || k == "comment" || k == "$db" {
-			continue
+	iter := resDB.Iterator()
+
+	for {
```

Comment:
I'm skeptical about `doc.Set(k, item)` being a part of this cycle. If there is a duplicate in keys, it'll panic.
Do we consider that duplicates are not possible in such a case, or should we use something else instead of `doc.Set`?

Reply:
We iterate over the predefined document here and in `pg` handler, so I think it's safe to use `doc.Set` here

Reply:
Ok, sounds good to me!